crates/biome_js_analyze/src/semantic_analyzers/correctness/use_exhaustive_dependencies.rs

```diff
@@ -519,6 +529,32 @@ fn function_of_hook_call(call: &JsCallExpression) -> Option<JsSyntaxNode> {
     })
 }
 
+// check if identifier is declared outside of function scope
+fn is_out_of_function_scope(
```

Comment:
```suggestion
/// Check if `identifier` is declared outside of `function` scope
fn is_out_of_function_scope(
```

Comments with triple slashes become "doc comments" and are nicely rendered in IDEs. They accept markdown :)

Reply:
Oh nice, will read up a bit about that